todo_app/app.py

```diff
@@ -2,19 +2,43 @@
 from todo_app.data.session_items import add_item, get_items
 
 from todo_app.flask_config import Config
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
+
+    url = "https://api.trello.com/1/boards/AbpSrXDQ/lists"
+
+    os.getenv("TRELLO_API_KEY")
+
+    querystring = {
+    "key":os.getenv("TRELLO_API_KEY"),
+    "token":os.getenv("TRELLO_API_TOKEN"),
+    "cards": "open" }
+
+    response = requests.request("GET", url, params=querystring)
+
+    response_json = response.json()
+
+    items = []
+
+    for trello_list in response_json:
+        for card in trello_list['cards']:
+            card['status'] = trello_list['name']
+            items.append(card)
+
     return render_template('index.html', items = items)
 
 @app.route('/create-a-todo', methods=['POST'])
 def create_new_todo():
     new_todo_title = request.form['todo-name']
     add_item(new_todo_title)
```

Comment:
Have you noticed that when you create to-do cards, they don't appear? You are still using the `add_item` function from session_items.py. So you are not adding anything to Trello.

You have written a good looking `create_card` function but you aren't using it yet!
- Import it (e.g. add this to the top of app.py: `import trello_api_call`)
- Use it (e.g. `trello_api_call.create_card(os.getenv('TODO_LIST_ID'), new_todo_title)`